frontend/src/libs/components/customer-card/customer-card.tsx

```diff
@@ -0,0 +1,26 @@
+import '~/assets/css/typography.scss';
```

Comment:
Isn't it imported to the main scss file? I think it is redundant

Reply:
yes, you are right. removed it